README.md

```diff
@@ -173,6 +154,28 @@ java \
 -javaagent:honeycomb-opentelemetry-javaagent-0.5.0-all.jar -jar myapp.jar
 ```
 
+### Multi-span Attributes
```

Comment:
Yes. I was trying to make it clear that multi-span attributes are available with the automatic BaggageSpanProcessor even without using the SDK Builder. So I have two separate sections - the first explains it, the second reminds you to initialize the builder... though it may not be necessary after your other PR.